app/services/application_state_change.rb

```diff
@@ -61,6 +61,7 @@ def initialize(application_choice)
       event :accept_unconditional_offer, transitions_to: :recruited
       event :decline, transitions_to: :declined
       event :decline_by_default, transitions_to: :declined
+      event :confirm_conditions_met, transitions_to: :recruited
```

Comment:
This fixes the problem, but I'm struggling to figure out why [this test](https://github.com/DFE-Digital/apply-for-teacher-training/blob/6585685de089aa0978c2121deec1da9b631918ef/spec/system/support_interface/change_offer_conditions_spec.rb#L3) wasn't failing.

Reply:
I was hoping we could hide the link in the support interface instead if it's in an offer condition as the provider can amend conditions themselves in manage as suggested in the ticket? 

Adding this extra transition in the offer state is not possible as we cannot move the application to a recruited state before an applicant has accepted the offer first.

The two scenarios for an offered application are:
- Unconditional offer: 
offer state -> candidate accepts -> recruited
- Conditional offer:
offer state -> candidate accepts -> conditions pending -> conditions confirmed by provider -> recruited

Reply:
ah ok, understood, thanks.